api/data.go

```diff
@@ -20,13 +23,29 @@ type dataLookupResponse struct {
 
 var dataTemplates *template.Template
 
-// SetupTimeseries loads templates for use by the timeseries lookup handler and should be done only once
+// SetupData loads templates for use by the timeseries lookup handler and should be done only once
 func SetupData() error {
 	templates, err := template.ParseFiles("templates/data/queryByUri.tmpl")
 	dataTemplates = templates
 	return err
 }
 
+func formatMultiQuery(rawQuery []byte) ([]byte, error) {
```

Comment:
I anticipate that the "data" endpoint will be replaced at some point as I integrate the service with Babbage. It will either be got rid of or will be refactored into the query builder package in the same way as I have done with the search endpoint. I was in two minds whether to do the refactoring straight away or to do it afterwards.

Reply:
I've implemented what was suggested.